modules/lightning_telemetry/src/Event.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\lightning_telemetry;
+
+/**
+ * AmplitudeEvent.
+ */
+class Event {
+
+  /**
+   * The event type.
+   *
+   * @var string
+   */
+  protected $type;
+
+  /**
+   * An array of event data.
+   *
+   * This key value pair will be sent to Amplitude as event_properties.
+   *
+   * @var array
+   */
+  protected $data;
+
+  /**
+   * The Amplitude user id for the event.
```

Comment:
What is Amplitude? I think we might need more documentation around this. Also, "id" should be "ID" everywhere.